### PR TITLE
Style updates for iPad Mini

### PR DESCRIPTION
**Pull request by molllusk**
Each shop will have an iPad to complete stock reports. Clerks don't always have access to a phone at the shop. These updates improve the experience on the iPad.

### CODE REVIEW

File: app/assets/stylesheets/application/custom.scss [hunk 1]
```diff
@@ -314,6 +314,14 @@ td.inventory_failure {
   border-color: #62838D;
 }
 
+.restock-section {
```
Comment (arvel) [inline app/assets/stylesheets/application/custom.scss:317]: For these two margins, you could use the built in margin classes from the Boostrap css file in assets. Like mt-4 or mb-3. But, not a huge deal. Just for future styling needs you know there are built in margin classes.

File: app/assets/stylesheets/application/custom.scss [hunk 2]
```diff
@@ -322,13 +330,26 @@ td.inventory_failure {
...
+
+.restock-table th, td {
   text-align: center;
-  font-size: 12px !important;
+  font-size: 14px !important;
+  vertical-align: middle;
+}
+
+.restock-table td {
+  padding-top: 5px;
```
Comment (arvel) [inline app/assets/stylesheets/application/custom.scss:348]: Also can use built in padding classes here too.

**Review by arvel — APPROVED** [review]
Looks good. Only comments have to do with explicitly specifying margins, padding and alignment when there are already built in classes for that.

Not at all major and totally fine to leave as is.

File: app/assets/stylesheets/application/custom.scss (hunk 1 above)
Comment (molllusk) [reply app/assets/stylesheets/application/custom.scss:317]: Understood. I didn't want to mess up the other tables in the app. I'll refactor our table styles soon to be consistent.

Comment (arvel) [reply app/assets/stylesheets/application/custom.scss:317]: > Understood. I didn't want to mess up the other tables in the app. I'll refactor our table styles soon to be consistent.

The classes have nothing to do with any other tables. It is just about adding or removing padding or margin to any element.

File: app/assets/stylesheets/application/custom.scss [hunk 3]
```diff
@@ -360,8 +382,14 @@ td.inventory_failure {
     text-align: center;
   }
 
+  .ipad-header {
+    text-align: center;
```
[the remark this answers is not in the document]
Comment (molllusk) [reply app/assets/stylesheets/application/custom.scss:393]: Updated! Also cleaned up a couple minor flaws.

**Comment by molllusk** [comment]
Thanks @arvel. Merging.

**Pull request merged**